db/dcrsqlite/apisource.go

```diff
@@ -1049,7 +1049,9 @@ func (db *wiredDB) GetExplorerAddress(address string, count, offset int64) *expl
 		int(offset), int(count), true, true, nil)
 	if err != nil {
 		log.Warnf("GetAddressTransactionsRaw failed for address %s: %v", addr, err)
-		return nil
+		return &explorer.AddressInfo{
+			Address: address,
```

Comment:
valid addresses for the wrong network will make it to here.  You can use `addr.IsForNet(db.params)` to check, but you might want to make a "ValidateNetworkAddress(string, *chaincfg.Params)` function or something like it.  Otherwise you don't really know why `GetExplorerAddress` returned `nil`

Reply:
What are the major distinctions for detection of a valid network address? for the function ValidateNetworkAddress(string, *chaincfg.Params)

Reply:
Each network had its own values for chaincfg.Params

Reply:
Yes, I mostly meant. where can i find the decred specific values for chaincfg.Params

Reply:
By network I mean the 3 Decred networks: mainnet, testnet, and simnet:  https://godoc.org/github.com/decred/dcrwallet/netparams

Reply:
Set in `wiredDB` on constructions: https://github.com/McEdward/dcrdata/blob/86c601d031ce647d6aa33b7a6ce21eec124ca16b/db/dcrsqlite/apisource.go#L40

Reply:
So in this case I'd be making sure it's on the mainnet

Reply:
Well it's supposed to work with any of them.  When the program starts up, the user sets the network and then `wiredDB.params` is set for you.  We just need to use that variable to validate for the network that dcrdata is running on.